### PR TITLE
Add Gnosis Network

### PR DESCRIPTION
**Pull request by Ajaypathak372**
## Changes:
- Gnosis network support
## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [x] Yes
- [ ] No

**In case you checked yes, did you write tests?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: templates/envs/gnosis/consensus/lighthouse.tmpl [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# --- Consensus Layer - Beacon Node - configuration ---
+CC_PEER_COUNT=50
+CC_LOG_LEVEL=info
+EC_API_URL={{.ExecutionApiURL}}
+EC_AUTH_URL={{.ExecutionAuthURL}}
+CC_INSTANCE_NAME=Lighthouse
+CC_IMAGE_VERSION={{if .CcImage }}{{.CcImage}}{{else}}sigp/lighthouse{{end}}
+CC_DATA_DIR={{.CcDataDir}}
+CC_JWT_SECRET_PATH={{.JWTSecretPath}}
+CL_FEE_RECIPIENT={{.FeeRecipient}}
```
Comment (cbermudez97) [inline templates/envs/gnosis/consensus/lighthouse.tmpl:12]: Please add the CL_BOOTNODES var for lighthouse template.

**Review by cbermudez97 — APPROVED** [review]

File: templates/envs/gnosis/consensus/lighthouse.tmpl (hunk 1 above)
Comment (Ajaypathak372) [reply templates/envs/gnosis/consensus/lighthouse.tmpl:12]: Done

**Review by AntiD2ta — APPROVED** [review]

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Please update the CHANGELOG file. The rest looks good to me

**Review by Ajaypathak372 — COMMENTED** [review]
Changelog updated

**Review by AntiD2ta — APPROVED** [review]

**Pull request merged**